src/server/api.js

```diff
@@ -0,0 +1,72 @@
+'use strict'; // eslint-disable-line
+
+const fs = require('fs');
+const soap = require('soap');
+const debug = require('debug')('hca:api');
+const ajv = require('ajv');
+
+const ApplicationJsonSchema = require('../common/schema/application');
+const validate = ajv({ allErrors: true, errorDataPath: 'property', removeAdditional: true, useDefaults: true }).compile(ApplicationJsonSchema);
+const veteranToSaveSubmitForm = require('./enrollment-system').veteranToSaveSubmitForm;
+const config = require('../../config');
+
+function readSecurityArtifacts() {
+  try {
+    fs.accessSync(config.soap.clientCertPath, fs.R_OK);
+    fs.accessSync(config.soap.clientKeyPath, fs.R_OK);
+    return {
+      scheme: 'ClientSSL',
+      certPath: config.soap.clientCertPath,
+      keyPath: config.soap.clientKeyPath
+    };
+  } catch (ex) {
+    debug("Can't read security artifacts. Starting without certificate and key.");
+    return undefined;
+  }
+}
+
+const securityArtifacts = readSecurityArtifacts();
+let voaService = null;
+soap.createClient(
+  config.soap.wsdl,
+  {
+    security: securityArtifacts,
+    wsdl_options: securityArtifacts === null ? null : { // eslint-disable-line
+      rejectUnauthorized: false,
+      strictSSL: false,
+      requestCert: true
+    }
+  },
+  (err, client) => { voaService = client; });
```

Comment:
we should deal with the error if we try to spin up and the server is offline it will blow up